GliaWidgets/Sources/CallVisualizer/ScreenSharing/ViewController/ScreenSharingViewController.swift

```diff
@@ -1,38 +1,41 @@
 import UIKit
+import SwiftUI
 
 extension CallVisualizer {
     final class ScreenSharingViewController: UIViewController {
-        private lazy var screenSharingView = ScreenSharingView(props: props.screenSharingViewProps)
-        private var props: Props
+        let model: ScreenSharingView.Model
 
         // MARK: - Initialization
 
-        init(props: Props) {
-            self.props = props
-            super.init(nibName: "", bundle: nil)
+        init(
+            model: ScreenSharingView.Model
+        ) {
+            self.model = model
+            super.init(nibName: nil, bundle: nil)
         }
 
+        @available(*, unavailable)
         required init?(coder: NSCoder) {
             fatalError("init(coder:) has not been implemented")
         }
 
         // MARK: - View lifecycle
 
-        override func loadView() {
-            view = screenSharingView
-        }
-
         override func viewDidLoad() {
             super.viewDidLoad()
-            renderProps()
+            let hostingController: UIHostingController<ScreenSharingView>
+            let screenSharingView = ScreenSharingView(model: model)
+            hostingController = UIHostingController(rootView: screenSharingView)
+            addChild(hostingController)
+            view.addSubview(hostingController.view)
```

Comment:
Just to clarify. Can we not to add `hostingController.view` as subview, but force it as root view of `ScreenSharingViewController` in `loadView()` method?